clang/lib/CIR/Dialect/Transforms/TargetLowering/LowerFunction.cpp

```diff
@@ -198,6 +198,23 @@ Value emitAddressAtOffset(LowerFunction &LF, Value addr,
   return addr;
 }
 
+mlir::cir::AllocaOp findAlloca(Operation *op) {
```

Comment:
I have the feeling you already introduced this function in another PR, seems like we need some reuse to happen?